packages/flutter_availability/lib/src/ui/widgets/defaults/default_buttons.dart

```diff
@@ -179,6 +179,8 @@ class DefaultSmallTextButton extends StatelessWidget {
         style: TextButton.styleFrom(
           textStyle: const TextStyle(
             decoration: TextDecoration.underline,
+            fontSize: 18,
```

Comment:
What's the benefit of updating the default button?

Reply:
not having to override it in beep